test.ts

```diff
@@ -76,27 +78,22 @@ export const execContainer = async (
   };
 };
 
+type TerraformStateResource = {
+  type: string;
+  name: string;
+  provider: string;
+  instances: [{ attributes: Record<string, any> }];
+};
+
 export interface TerraformState {
   outputs: {
     [key: string]: {
       type: string;
       value: any;
     };
-  }
-  resources: [
-    {
-      type: string;
-      name: string;
-      provider: string;
-      instances: [
-        {
-          attributes: {
-            [key: string]: any;
-          };
-        },
-      ];
-    },
-  ];
+  };
+
+  resources: [TerraformStateResource, ...TerraformStateResource[]];
```

Comment:
Redefined this so that `resources` is an array with at least element, rather than an array with exactly one element. This typo was causing some of the other tests to do stuff with the `any` type; I'll be making a separate PR to clean up more stuff